packages/cli/src/commands/index.ts

```diff
@@ -1,18 +1,19 @@
 import { CommandModule } from "yargs";
 
-import bulkupload from "./bulkupload.js";
-import callSystem from "./call-system.js";
-import codegenLibdeploy from "./codegen-libdeploy.js";
-import deployContracts from "./deploy-contracts.js";
+import bulkupload from "./deprecated/bulkupload.js";
+import callSystem from "./deprecated/call-system.js";
+import codegenLibdeploy from "./deprecated/codegen-libdeploy.js";
+import deployContracts from "./deprecated/deploy-contracts.js";
+import systemTypes from "./deprecated/system-types.js";
+import test from "./deprecated/test.js";
+import trace from "./deprecated/trace.js";
+import types from "./deprecated/types.js";
```

Comment:
I don't think I noticed this before but why the explicit `.js` in imports?

Reply:
I'm being compatible with `moduleResolution="nodenext"`. In a sense this could be run in a browser (maybe, eventually)

Reply:
basically this - https://nodejs.org/api/esm.html#mandatory-file-extensions